fin-wallet-api/src/main/java/com/codegym/finwallet/controller/AppUserController.java

```diff
@@ -2,24 +2,34 @@
 
 import com.codegym.finwallet.dto.CommonResponse;
 import com.codegym.finwallet.dto.payload.request.UpdateProfileRequest;
+import com.codegym.finwallet.service.ImageService;
 import com.codegym.finwallet.service.ProfileService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PatchMapping;
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
 
 @RestController
 @RequiredArgsConstructor
 @RequestMapping("/api/v1/user")
 public class AppUserController {
 
     private final ProfileService profileService;
+    private final ImageService imageService;
 
     @PutMapping("/update-profile")
```

Comment:
this is PUT method, we should understand is update action => rename /update-profile to /profile